tests/PowerShellTemplate.Tests.ps1

```diff
@@ -0,0 +1,16 @@
+﻿[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSAvoidGlobalVars", "")]
```

Comment:
This file could use some comments explaining why it exists, and what it's doing and why it's positioned where it is.

Reply:
Removed it, as unrequired. Apologies. Also removed from CloneModule.